apps/console/src/features/identity-providers/components/forms/authenticators/apple-authenticator-form.tsx

```diff
@@ -0,0 +1,660 @@
+                    ".authenticatorSettings.apple.scopes.list.email.description"),
+                displayName: (
+                    <Code compact withBackground={ false } fontSize="inherit" fontColor="inherit">
+                        { IdentityProviderManagementConstants.APPLE_SCOPE_DICTIONARY.EMAIL }
+                    </Code>
+                ),
+                icon: "envelope outline"
+            };
+        }
+
+        if (scope === IdentityProviderManagementConstants.APPLE_SCOPE_DICTIONARY.NAME) {
+            return {
+                description: t("console:develop.features.authenticationProvider.forms" +
+                    ".authenticatorSettings.apple.scopes.list.name.description"),
+                displayName: (
+                    <Code compact withBackground={ false } fontSize="inherit" fontColor="inherit">
+                        { IdentityProviderManagementConstants.APPLE_SCOPE_DICTIONARY.NAME }
+                    </Code>
+                ),
+                icon: "user outline"
+            };
+        }
+
+        return {
+            description: "",
+            displayName: scope,
+            icon: "key"
+        };
+    };
+
+    return (
+        <Form
+            id={ FORM_ID }
+            uncontrolledForm={ false }
+            onSubmit={ (values: Record<string, any>) => onSubmit(getUpdatedConfigurations(values as any)) }
+            initialValues={ initialValues }
+        >
+            <Field.Input
+                ariaLabel="Apple authenticator client ID"
+                inputType="default"
+                name="ClientId"
+                label={
+                    t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +
+                        ".apple.clientId.label")
+                }
+                placeholder={
+                    t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +
+                        ".apple.clientId.placeholder")
+                }
+                hint={
+                    t("console:develop.features.authenticationProvider.forms.authenticatorSettings" +
+                        ".apple.clientId.hint")
+                }
+                required={ formFields?.ClientId?.meta?.isMandatory }
+                readOnly={
+                    readOnly || (
+                        mode === AuthenticatorSettingsFormModes.CREATE
+                            ? false
+                            : formFields?.ClientId?.meta?.readOnly
+                    )
+                }
+                value={ formFields?.ClientId?.value }
+                maxLength={ formFields?.ClientId?.meta?.maxLength }
+                minLength={
+                    IdentityProviderManagementConstants
+                        .AUTHENTICATOR_SETTINGS_FORM_FIELD_CONSTRAINTS.CLIENT_ID_MIN_LENGTH as number
+                }
+                width={ 16 }
+                data-testid={ `${ componentId }-client-id` }
```

Comment:
Can we update this to data-componentid?

Reply:
This is an existing component that doesn't have the data-componentId. We have to pass test Ids to them?